plugin/PDFDocumentPage.qml

```diff
@@ -44,6 +45,11 @@ DocumentPage {
     PDF.Document {
         id: pdfDocument;
         source: base.path;
+        onDocumentLoaded: if (failure) {
+            pageStack.completeAnimation();
+            pageStack.pop();
+            notification.publish();
```

Comment:
Yep, that's great of you. Currently the page is pushed and then the PDF document is loaded, so I don't know how not to push the page without being too intrusive in the code.

Besides, this may be irrelevant not to push the page because, I'm preparing a kind of "place holder" for PDF without rendering (because of broken file or because the file is valid but currently locked down). This place holder would replace the PDFView Item and display a message that the PDF is either broken or protected. In case of protected file a button would open a dialog with a text input to enter the password (related to some discussion in issue #1). Having this place holder would keep the actions of the drawer available, like sharing the file by bluetooth…

What do you think ?